otcextensions/sdk/dds/v3/instance.py

```diff
@@ -132,7 +188,40 @@ class Instance(resource.Resource):
     backup_strategy = resource.Body('backup_strategy', type=BackupStrategySpec)
     #: Specifies whether to enable SSL.
     #: *Type:string*
-    ssl_option = resource.Body('ssl_option')
+    ssl = resource.Body('ssl')
```

Comment:
Can you please sort attributes alphabetically. Can be done also in followup

Reply:
sorted